harness/determined/workload.py

```diff
@@ -30,8 +40,12 @@ def __hash__(self) -> int:
         return hash((self.kind, self.experiment_id, self.trial_id, self.step_id))
 
     def __repr__(self) -> str:
-        return "<{}: ({},{},{})>".format(
-            self.kind.name, self.experiment_id, self.trial_id, self.step_id
+        return "<{}{}: ({},{},{})>".format(
+            self.kind.name,
+            " ( {})".format(self.batches_per_step) if self.kind == self.Kind.RUN_STEP else "",
```

Comment:
Switch to `f""` strings in the harness code.  These `.format()` calls are really old.  I think we still support python 3.5 in the CLI code, but in the harness anymore.

Separately, probably just make this line a variable, I had to do a double take to figure out what was going on.

Lastly, it seems like there's an extra space after the `(` character of this format string... is that intentional?

Reply:
Ah yeah, I've usually been stuck in python3.5 world. The extra space isn't intentional, I have it fixed in my local, sorry.